### PR TITLE
Show incoming mail configuration on dashboard

### PR DESCRIPTION
**Pull request by hiifong**
Preparing for implementation of #32665

image

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -3289,6 +3289,18 @@ config.send_test_mail_submit = Send
...
+config.incoming_mail_enabled = Enabled
+config.incoming_mail_reply_to_address = Reply To Address
+config.incoming_mail_host = Host
+config.incoming_mail_port = Port
+config.incoming_mail_username = Username
+config.incoming_mail_password = Password
+config.incoming_mail_use_tls = Use TLS
+config.incoming_mail_skip_tls_verify = Skip TLS Verify
+config.incoming_mail_mailbox = Mailbox
+config.incoming_mail_maximum_message_size = Maximum Message Size
```
Comment (wxiaoguang) [inline options/locale/locale_en-US.ini]: I do not think it is good to translate all these config options.

There are hundreds of them, who would really translate them one by one?

File: templates/admin/config.tmpl [hunk 2]
```diff
@@ -243,6 +243,34 @@
...
+				<dt>{{ctx.Locale.Tr "admin.config.incoming_mail_reply_to_address"}}</dt>
+				<dd>{{.IncomingMail.ReplyToAddress}}</dd>
+				<dt>{{ctx.Locale.Tr "admin.config.incoming_mail_host"}}</dt>
+				<dd>{{.IncomingMail.Host}}</dd>
+				<dt>{{ctx.Locale.Tr "admin.config.incoming_mail_port"}}</dt>
+				<dd>{{.IncomingMail.Host}}</dd>
+				<dt>{{ctx.Locale.Tr "admin.config.incoming_mail_username"}}</dt>
+				<dd>{{.IncomingMail.Host}}</dd>
+				<dt>{{ctx.Locale.Tr "admin.config.incoming_mail_password"}}</dt>
+				<dd>{{.IncomingMail.Password}}</dd>
```
Comment (wxiaoguang) [inline templates/admin/config.tmpl]: The password should never be shown in cleartext.

**Comment by wxiaoguang** [comment]
TBH I do not think keeping adding "config options display" would really help end users, there are too many of them. That page was already a mess.

**Comment by hiifong** [comment]
Because smtp has no way of knowing if a message is bounced, I can't proceed with implementing the feature

**Pull request closed without merging**